src/main/java/bio/terra/dao/StudyDao.java

```diff
@@ -56,38 +43,32 @@ public UUID create(Study study) {
 
     @Transactional
     public boolean delete(UUID id) {
-        int rowsAffected = jdbcTemplate.update("DELETE FROM study WHERE id = :id",
+        int rowsAffected = getJdbcTemplate().update("DELETE FROM study WHERE id = :id",
```

Comment:
it can, but it makes it a little awkward for the super class. i initially had it as protected in the superclass so the subclass could reference it directly, but one of the code checkers didn't like that. it wanted it to be made private and to add a getter. maybe that is a check we don't need.

if we want to keep that check in and have a private variable for both classes, the subclass would be passing it in to the super constructor and storing it locally. which looks a bit strange.

Reply:
Ah yes, I've hit that check writing tests.
I have no problem leaving this just how it is.